crates/but-cli/src/args.rs

```diff
@@ -31,6 +31,11 @@ pub enum Subcommands {
         /// The message of the new commit.
         #[clap(long, short = 'm')]
         message: Option<String>,
+        /// The name of the reference that the commit should be in.
+        ///
+        /// If there is ambiguity, this is what makes it ambiguous.
+        #[clap(long, short = 's')]
+        stack_segment_ref: Option<String>,
```

Comment:
@krlvi I'm wondering if `stack_segment` is a term we want to introduce? To me it would feel more natural to talk about branches.